cordformation/build.gradle

```diff
@@ -3,6 +3,12 @@ apply plugin: 'java-gradle-plugin'
 
 description 'A small gradle plugin for adding some basic Quasar tasks and configurations to reduce build.gradle bloat.'
 
+buildscript {
```

Comment:
No need for `buildscript` here. This will suffice:
```gradle
ext {
    corda_release_version = '4.0'
}
```

Reply:
Fixed c8e9113b97ccbd98cc8e9bf4492320a919d058ca